app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java

```diff
@@ -154,12 +160,13 @@ private void storeStudentImage(Mat img){
 
         MatName matName = new MatName(sId, img);
         FileHelper fh = new FileHelper();
-        String wholeFolderPath = MultimediaHelper.getStudentImageDirectory() + "/" + deviceId + "/" + collectionEventId;
+        String wholeFolderPath = MultimediaHelper.getStudentImageDirectory() + "/" + deviceId + "/" + sId;
         new File(wholeFolderPath).mkdirs();
         fh.saveMatToImage(matName, wholeFolderPath + "/");
 
         Long Id =  Long.parseLong(String.valueOf((int) (Math.random() * 1000000)));
         StudentImage studentImage = new StudentImage(Id, null, wholeFolderPath, null, null);
```

Comment:
- Haven't been sure about the DeviceID yet but will convert it to Hex now and save it in a long.
- @jogrimst how should we store the time? With help of the CalendarConverter? Do you might have an example? 
- Student Image Feature is pending